architecture-docs/firebase-architecture.md

```diff
@@ -0,0 +1,12 @@
+The Ground platform uses Firebase for many backend and utility operations, including authentication and storage. 
+
+Currently, Ground uses the following Firebase features:
+
+- [**Cloud Firestore**](https://firebase.google.com/docs/firestore) is used for storing Ground observation data. Data in Firestore is organized into collections and documents. For more information on how Ground data is structured, see [Cloud Firestore Representation](https://github.com/google/ground-platform/wiki/Cloud-Firestore-Representation).
+- [**Cloud Function for Firebase**](https://firebase.google.com/docs/functions) are used for two features: 
```

Comment:
Function->Functions

Reply:
What's listed here are not features of Ground, but rather two types of Firestore Cloud Functions; it might be more useful to list the Ground features these are used for:
* Data import and export
* Database triggers used to synchronize data to Google Sheets (TBD).

Reply:
Done.